.github/workflows/tests.yml

```diff
@@ -167,8 +167,10 @@ jobs:
     - name: Install Ghidra 10.3
       run: |
         mkdir ./.github/ghidra
+        mkdir ./.github/ghidra/project
         wget "https://github.com/NationalSecurityAgency/ghidra/releases/download/Ghidra_10.3_build/ghidra_10.3_PUBLIC_20230510.zip" -O ./.github/ghidra/ghidra_10.3_PUBLIC.zip
         unzip .github/ghidra/ghidra_10.3_PUBLIC.zip -d .github/ghidra/
+        .github/ghidra/ghidra_10.3_PUBLIC/ghidraRun 
```

Comment:
I'm concerned with maintaining all of the hard-coded `ghidra_10.3` values. What happens with a new release of Ghidra? Should we consider always testing against the latest Ghidra release?

Reply:
maybe we could use a matrix of version strings, like we do here: https://github.com/mandiant/capa/blob/43b2ee3c5281f3ec0f3834a34769e7e10af9ede3/.github/workflows/tests.yml#L69-L80

and interpolate the zip/directory names like we do here: https://github.com/mandiant/capa/blob/43b2ee3c5281f3ec0f3834a34769e7e10af9ede3/.github/workflows/build.yml#L108

then its only one place to update when a new release comes up. maybe keeping around the last 3 or so releases is reasonable?

Reply:
I've updated the config to include the rest of the versions for the dependencies in a matrix. I believe that this was a good move; however, we may have to pick-and-choose which dependencies specifically to test on multiple versions. I'm not quite sure how github's runner will handle the CI with so much to download and install.

However, if it's no issue, we can quite easily test against multiple versions of all dependencies.